src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.cs

```diff
@@ -77,27 +77,27 @@ public static ReadOnlySpan<byte> AsBytes<T>(ReadOnlySpan<T> span)
         /// Returns a reference to the 0th element of the Span. If the Span is empty, returns a reference to the location where the 0th element
         /// would have been stored. Such a reference may or may not be null. It can be used for pinning but must never be dereferenced.
         /// </summary>
-        public static ref T GetReference<T>([LifetimeAnnotation(false, true)] Span<T> span) => ref span._reference.Value;
+        public static ref T GetReference<T>(Span<T> span) => ref Unsafe.AsRef<T>(in span._reference);
```

Comment:
Why do these calls need to use Unsafe.AsRef?

Reply:
Ah right. Now that `scoped` is removed, these should also be removed.